src/components/SectionForm/SectionForm.tsx

```diff
@@ -29,20 +29,30 @@ const submitTopics = (event:React.MouseEvent): void => {
 const sectionCheckboxes = topics.map(topic => {
   const key = Math.random();
   return (
-    <div className='article-section'>
-      <label>{topic}</label>
+    <span className="article-section">
       <input className="section" id={topic.toLowerCase()} type="checkbox" checked={selectedTopics.includes(topic) ? true : false} key={key} value={topic.toLowerCase()} onChange={() => handleChange(topic)}/>
-    </div>
+      <label className="topic-label" onClick={() => handleChange(topic)}>{topic}</label>
+    </span>
   )
 })
 
   return (
-    <form>
-    { sectionCheckboxes }
-      <button onClick={e => submitTopics(e)}>
-        Filter
-      </button>
-    </form>
+    <div className="filter-container">
+      <form className="filter-options hidden">
+        <div className="checkbox-container-1 checkbox-container">
```

Comment:
This works for now, but we are going to add the number of sections dynamically and we will not always have the same sections/topics available so we need a more dynamic way to add them to the page... maybe using flex wrap would make sure they wrap around when the container ends?